app/assets/images/facebook.svg

```diff
@@ -1 +1 @@
-<svg width="24" height="24" xmlns="http://www.w3.org/2000/svg"><path d="M15.402 21v-6.966h2.333l.349-2.708h-2.682V9.598c0-.784.218-1.319 1.342-1.319h1.434V5.857a19.188 19.188 0 00-2.09-.107c-2.067 0-3.482 1.262-3.482 3.58v1.996h-2.338v2.708h2.338V21H4a1 1 0 01-1-1V4a1 1 0 011-1h16a1 1 0 011 1v16a1 1 0 01-1 1h-4.598z" /></svg>
+<svg width="24" height="24" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M15.402 21v-6.966h2.333l.349-2.708h-2.682V9.598c0-.784.218-1.319 1.342-1.319h1.434V5.857a19.188 19.188 0 00-2.09-.107c-2.067 0-3.482 1.262-3.482 3.58v1.996h-2.338v2.708h2.338V21H4a1 1 0 01-1-1V4a1 1 0 011-1h16a1 1 0 011 1v16a1 1 0 01-1 1h-4.598z" /></svg>
```

Comment:
Without the viewbox, the css on the elements aren't resizing the width and height of the svg as per my PR description

Reply:
I wonder if we should add a comment there, as that if the icons are ever updated we'll probably won't notice `viewBox` disappearing

Reply:
@rhymes usually whenever I've downloaded an svg it has a viewbox attribute, its pretty standard. I'm not sure why these didn't have them. It seems like a bit of an anomaly.

Reply:
@rhymes 
yea - i blame myself for that.. i'll go through all our SVG icons and add viewBox attribute but in a separate PR :)